### PR TITLE
Add event to blocks which was placed by the bot

### PR DESCRIPTION
**Pull request by CyberPatrick**

### CODE REVIEW

File: lib/plugins/place_block.js [hunk 1]
```diff
@@ -73,6 +73,8 @@ function inject (bot, { version }) {
 
     if (oldBlock.type === newBlock.type) {
       throw new Error(`No block has been placed : the block is still ${oldBlock.name}`)
+    } else {
+      bot.emit('blockPlaced', [oldBlock, newBlock])
```
Comment (Karang) [inline lib/plugins/place_block.js:77]: You don't need to put the parameters between brackets, unless you want them to be a single array parameter.

**Review by Karang — CHANGES_REQUESTED** [review]
One comment, otherwise looks good to me. Can you add the event to api.md ?

File: lib/plugins/place_block.js (hunk 1 above)
Comment (CyberPatrick) [reply lib/plugins/place_block.js]: Ok

**Review by Karang — APPROVED** [review]
LGTM

**Pull request merged**